examples/oidc_cli/src/main.rs

```diff
@@ -466,7 +617,7 @@ impl OidcCli {
 
         oidc.finish_authorization(authorization_code).await?;
 
-        // Now we refresh the stored session to always have the latest tokens.
+        // Now we store the latest session to always have the latest tokens.
         self.update_stored_session().await?;
```

Comment:
It should be unnecessary here too.

Reply:
The API is a bit asymmetric here, but the previous code (and that I haven't changed) didn't send a session change broadcast upon login, only for token refresh, so we do need a manual call here at the moment.

Reply:
Ah right, I was thinking of `Oidc::session_tokens_stream`